src/assets/scripts/app.js

```diff
@@ -0,0 +1,25 @@
+// Styles
+import '../styles/site.css';
+
+// Views
+import '../../views/scorer.ejs';
+import '../../views/rescue_scorer_2018_nacional.ejs';
+import '../../views/modal_select_team.ejs';
+
+// Scripts
+import './api';
+import './app1';
+import './app2';
+import './app3';
+import './app4';
```

Comment:
O que são "app1", ... "app4"? Melhor inserir o nome dos módulos se for isso mesmo (alterar o nome dos arquivos)

Reply:
Eram todos os arquivos que eu encontrei com o nome `app.js`. Eu ia alterar o nome quando visse para quê servia cada um, mas acabei esquecendo